public/plugins/highlight-9.1.0/github.css

```diff
@@ -7,9 +7,9 @@ github.com style (c) Vasily Polovnyov <vast@whiteants.net>
 .hljs {
   display: block;
   overflow-x: auto;
-  padding: 0.5em;
+  /*padding: 0.5em;*/
```

Comment:
I think overwrite this value in `_base.less` is better as we can't care very well when upgrading the plugins.